frontend/src/pages/Posts.jsx

```
@@ -0,0 +1,326 @@
+    if (search.trim() !== "") {
+      const searchLower = search.toLowerCase();
+      filtered = filtered.filter(post => 
+        post.name?.toLowerCase().includes(searchLower) ||
+        post.position?.toLowerCase().includes(searchLower) ||
+        post.user?.name?.toLowerCase().includes(searchLower)
+      );
+    }
+
+    // Filter by salary range
+    if (salary.trim() !== "") {
+      const salaryValue = parseFloat(salary);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)